cmd/cli/trafficpolicy_check.go

```diff
@@ -13,14 +13,17 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
 
-	"github.com/openservicemesh/osm/pkg/configurator"
+	osmConfigClient "github.com/openservicemesh/osm/pkg/gen/client/config/clientset/versioned"
 )
 
 const trafficPolicyCheckDescription = `
 This command will check whether a given source pod is allowed to communicate
 (send traffic) to a given destination pod by an SMI TrafficTarget policy or
 in lieu of the mesh operating in permissive traffic policy mode.
+
+MESH_CONFIG_NAME is optional. The default value is "osm-mesh-config".
```

Comment:
When will this have a different name?

Reply:
I messed up. I thought in the current implementation, configmap name can be customized.

Will remove.